### ISSUE TITLE
Project Creation changes for latest Visual Studio Code (minor)

### ISSUE THREAD

**Issue by AnilDesai** [dotnet/AspNetCore.Docs#19432]
At least the first several steps for using VS Community appear to be outdated.  The setup options for a new Blazor project now uses two steps and offers more options.  For example, it's useful to disable HTTPS for learning purposes.
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 08de9daf-79aa-ee76-431e-f2432f8d691e
* Version Independent ID: 1a1ee728-1741-c82a-75e1-06159709935b
* Content: [Use ASP.NET Core SignalR with Blazor WebAssembly](https://docs.microsoft.com/en-us/aspnet/core/tutorials/signalr-blazor-webassembly?view=aspnetcore-3.1&tabs=visual-studio)
* Content Source: [aspnetcore/tutorials/signalr-blazor-webassembly.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/tutorials/signalr-blazor-webassembly.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

**Comment by guardrex** [comment]
Hello @AnilDesai ... I'm surprised. These instructions are correct for the commercial version, and I don't understand how the UI would be different. Are you able to take a screenshot of each step and place them into a comment here so that I can see? Alternatively, can you clarify exactly which steps aren't correct and what the UI shows instead? Installing VS Community Ed. isn't simple for me given my storage space limitations on this machine ... I've just about maxed out this old thing.

**Comment by AnilDesai** [comment]
Hi, Luke - thanks for your extremely quick response to my issue.  I believe the issue I encountered might have been due to either user error (me) or having an earlier version of the template installed.  Just to clarify, this was a minor problem (and one that I think most readers wouldn't have any problems with).  I did mean to state **VS Community** and am using the instructions for "Visual Studio" in the article (apologies for the confusion).  

I just retried the steps, and the documentation is exactly correct (no changes needed).  Attached are the screenshots of the two setup dialogs for creating a new project.  I had just installed VS Community 2019 within the last few days and verified that I'm using the latest update.  Again, sorry for the confusion and feel free to close the issue.
<img width="449" alt="VS Blazor App 1" src="https://user-images.githubusercontent.com/7740584/89712971-01e09180-d95a-11ea-94c2-76c63a896a09.png">
<img width="512" alt="VS Blazor App 2" src="https://user-images.githubusercontent.com/7740584/89712973-0311be80-d95a-11ea-9fb0-84e9855b13fc.png">
